toolkit/docs/building/dedicated_guides/build_packages.md

```diff
@@ -0,0 +1,163 @@
+# Building Packages in the Core Repo
+
+## Overview
+
+This document details building packages in the core repo. Most people should start with [CBL-MarinerTutorials](https://github.com/microsoft/CBL-MarinerTutorials). This is the correct way to build custom packages for CBL-Mariner. Continue here only if you care about building the `.spec` files present in the core repo.
```

Comment:
```suggestion
This document describes how to build packages in the core repo. Most people should start with [CBL-MarinerTutorials](https://github.com/microsoft/CBL-MarinerTutorials). This is the correct way to build custom packages for CBL-Mariner. Continue here only if you care about building the `.spec` files present in the core repo.
```

Reply:
Same questions here as in the images document.

Reply:
```suggestion
This document describes how to build packages in the core repo. Most people should start with [CBL-MarinerTutorials](https://github.com/microsoft/CBL-MarinerTutorials).

The following document shows the correct way to build packages for CBL-Mariner. Follow the instructions if you want to build the `.spec` files present in the core repo.
```